lib/bug_report_templates/rails_5_master.rb

```diff
@@ -0,0 +1,120 @@
+begin
+  require 'bundler/inline'
+rescue LoadError => e
+  STDERR.puts 'Bundler version 1.10 or later is required. Please update your Bundler'
+  raise e
+end
+
+gemfile(true) do
+  source 'https://rubygems.org'
+
+  gem 'rails', require: false
+  gem 'sqlite3', platform: :mri
+
+  gem 'activerecord-jdbcsqlite3-adapter',
+      git: 'https://github.com/jruby/activerecord-jdbc-adapter',
+      branch: 'rails-5',
+      platform: :jruby
+
+  if ENV['ACTIVE_ADMIN_PATH']
+    gem 'activeadmin', path: ENV['ACTIVE_ADMIN_PATH'], require: false
+  else
+    gem 'activeadmin', git: 'https://github.com/activeadmin/activeadmin', require: false
+  end
+
+  gem 'inherited_resources', '~> 1.7', require: false
+end
+
+# prepare active_record database
+require 'active_record'
+
+ActiveRecord::Base.establish_connection(adapter: 'sqlite3', database: ':memory:')
+ActiveRecord::Base.logger = Logger.new(STDOUT)
+
+ActiveRecord::Schema.define do
+  create_table :active_admin_comments, force: true do |_t|
+  end
+
+  # Add your schema here
+  create_table :your_models, force: true do |t|
+    t.string :name
+  end
+end
+
+# initialize app
+require 'action_controller/railtie'
+require 'action_view/railtie'
+require 'active_admin'
+
+class ApplicationController < ActionController::Base
+end
+
+class TestApp < Rails::Application
+  config.root = File.dirname(__FILE__)
+  config.logger = Logger.new(STDOUT)
+
+  secrets.secret_token = 'secret_token'
+  secrets.secret_key_base = 'secret_key_base'
+
+  config.eager_load = false
+  config.assets.precompile += %w( active_admin/print.css )
```

Comment:
Figured out how to remove this line.
`Rails.application.initialize!` should be places after `ActiveAdmin.setup` was called, because it adds default asset list to `ActiveAdmin.stylesheets` and `ActiveAdmin.javascripts`.
But rails app initializing should be placed before `ActiveAdmin` registers pages/resources to work correctly.

After all I just repeat loading order for rails app parts more precisely